en/02_Developer_Guides/15_Customising_the_Admin_Interface/06_Javascript_Development.md

```diff
@@ -56,71 +42,13 @@ You should write all your custom jQuery code in a closure.
 })(jQuery);
 ```
 
-## jQuery Plugins
+#### jQuery Plugins
 
 A jQuery Plugin is essentially a method call which can act on a collection of DOM elements. It is contained within the
-`jQuery.fn` namespace, and attaches itself automatically to all jQuery collections. The basics for are outlined in the
-official [jQuery Plugin Authoring](http://docs.jquery.com/Plugins/Authoring) documentation.
-
-There a certain [documented patterns](http://www.learningjquery.com/2007/10/a-plugin-development-pattern) for plugin
-development, most importantly:
-
-*  Claim only a single name in the jQuery namespace
-*  Accept an options argument to control plugin behavior
```

Comment:
Our docs don't need to explain what jQuery Plugins are good for or how to build them. Leave that to the official docs.